install.sh

```diff
@@ -97,6 +97,8 @@ if [ "$DISTRIBUTION" = "Darwin" ]; then
   OS="macOS"
 elif [ -f /etc/debian_version ] || [ "$DISTRIBUTION" == "Debian" ] || [ "$DISTRIBUTION" == "Ubuntu" ]; then
   OS="Debian"
+elif [ "$AWS_EXECUTION_ENV" == "CloudShell" ]; then
+  OS="CloudShell"
```

Comment:
should we name this "AWSCloudShell" since we probably will add support for GCP and Azure going forward